roles/core/cluster/tasks/cluster.yml

```diff
@@ -43,6 +43,20 @@
   with_items: "{{ ansible_play_hosts }}"
   changed_when: false
 
+- name: cluster | Group defined daemon network nodename
+  add_host:
+     name: "{{ item }}"
+     groups: scale_daemon_network_nodes
+  when: hostvars[item]['scale_daemon_network'] is defined
+  with_items: "{{ ansible_play_hosts }}"
+
+- assert:
+    that: ansible_play_hosts|length == groups['scale_daemon_nodes']|length
+    msg: >-
+         Please specify the daemon network for all nodes in the cluster.
+  when: groups['scale_daemon_nodes'] is defined
+  run_once: true
```

Comment:
Does this mean that `scale_daemon_network` becomes a mandatory variable, in all cases?

Reply:
Hi Achim,
sorry for my late response but I had a system crash on my laptop. 
Sure we have to update the documentation with that new feature. I will discuss this with Rajan.  

The scale_daemon_network is a optional variable that will be set at the inventory file on each cluster host
If a daemon network should be setup the customer has to specify the name of the network alias for the daemon network ip.

Reply:
Thanks for this explanation.

I understand that `scale_daemon_network` is optional. However, in this code block you're testing (asserting) that all hosts in the current play have this var defined... and that makes the var a mandatory one... or am I missing something?